typst: add page (#10497)
tldr-pages/tldr: pages/common/typst.md

# typst

> > Compile a typst file to PDF.

Reviewer: ```suggestion
> Compile a Typst file to PDF.
> Note: Specifying output location is optional.
```

> More information: <https://github.com/typst/typst>.

- List all discoverable fonts in the system and the given directory:

> `typst --font-path {{path/to/fontDir}} fonts`

Reviewer: ```suggestion
`typst --font-path {{path/to/fonts_directory}} fonts`
```

> - Compile a Typst file directly:
> 
> `typst compile {{path/to/source.typ path/to/output.pdf}}`

Reviewer: ```suggestion
- Compile a Typst file directly:

`typst compile {{path/to/source.typ}} {{path/to/output.pdf}}`
```

- Watch changes on a Typst file and compile dynamically:

> `typst watch {{path/to/source.typ path/to/output.pdf}}`

Reviewer: ```suggestion
`typst watch {{path/to/source.typ}} {{path/to/output.pdf}}`
```

# typst

> Compile a Typst file to PDF.

> > Note: Specifying output location is optional.

Reviewer 2: ```suggestion
> Note: Specifying the output location is optional.
```

> More information: <https://github.com/typst/typst>.

- List all discoverable fonts in the system and the given directory:

`typst --font-path {{path/to/fonts_directory}} fonts`

> - Compile a Typst file directly:

Reviewer 2: ```suggestion
- Compile a Typst file:
```

`typst compile {{path/to/source.typ}} {{path/to/output.pdf}}`

> - Watch changes on a Typst file and compile dynamically:

Reviewer 2: ```suggestion
- Watch a Typst file and recompile on changes:
```
